src/js/plugins/plugin.mobile_nav.js

```diff
@@ -70,9 +77,32 @@ BookReader.prototype.initToolbar = (function (super_) {
         }
       });
 
-      // High contrast button
-      $drawerEl.find('.high-contrast-button').click(
-        () => this.refs.$br.toggleClass('high-contrast'));
+
+      // Dinamically creates styles combining different filters for BookReaders imgs
+      // based on filters checkbox
+      $drawerEl.find('.BRcheckbox-filters').click(
+        () => {
+          const br = this;
+          $("#filtersStyle").remove();
+          let filterStr = "";
+          let filterWebkitStr = "";
+
+          $('.BRcheckbox-filters').each(
+            function () {
```

Comment:
```suggestion
          $('.BRcheckbox-filters').each(
            (i, el) => {
```

That way we don't need the `const br = this;`